test/contrib/stun.uts

```diff
@@ -32,7 +32,7 @@ assert parsed.attributes == [
       STUNPriority(priority=1847591167),
       STUNIceControlling(tie_breaker=0x1b0ab98b6e8effa6),
       STUNUsername(length=37, username="oNph:Ht11MaRZHc4GOLJUsbu1R3YCs72HYN25"),
-      STUNMessageIntegrity(hmac_sha1=0x77e7f03e7f9e1996be42339159db1f682147bcfc),
+      STUNMessageIntegrity(hmac_sha1=0xfcbc4721681fdb59913342be96199e7f3ef0e777),
```

Comment:
`raw` contains
```
\xfc\xbc\x47\x21" \
      b"\x68\x1f\xdb\x59\x91\x33\x42\xbe\x96\x19\x9e\x7f\x3e\xf0\xe7\x77
```
so I'm not sure why the test expected
```
0x77e7f03e7f9e1996be42339159db1f682147bcfc
```
instead of
```
0xfcbc4721681fdb59913342be96199e7f3ef0e777
```
here. My guess would be that it was just copied verbatim from `STUN(raw).attributes` at the time.